native/acter/src/api/stream.rs

```diff
@@ -411,7 +412,7 @@ impl TimelineStream {
 
     pub async fn retry_send(&self, txn_id: String) -> Result<bool> {
         let timeline = self.timeline.clone();
-        let transaction_id = OwnedTransactionId::from(txn_id);
```

Comment:
this one I don't understand. Why replace a never failing `from` with a sometimes failing `try_from(...)?`?

Reply:
I just wanted to capture incorrect usage in development stage.

Reply:
And I don't know why both `from()` and `try_from()` exist in `OwnedTransactionId`.
If failure is possible like `OwnedEventId` or `OwnedUserId`, `from()` should not exist.